Makefile

```diff
@@ -135,6 +135,8 @@ test-python-ci: create-db
 		--ignore tests/sentry/ingest/ingest_consumer/test_ingest_consumer_kafka.py \
 		--ignore tests/sentry/region_to_control/test_region_to_control_kafka.py \
 		--cov . --cov-report="xml:.artifacts/python.coverage.xml"
+	SENTRY_USE_MONOLITH_DBS=1 pytest tests/sentry/backup \
+		--cov . --cov-report="xml:.artifacts/python-backup-tests.coverage.xml"
```

Comment:
we've been working hard to *reduce* the number of special ways to run our testsuite.  I would strongly prefer this be done inside the test and not in the Makefile

it's been a constant source of feedback in dev-infra surveys that running ci tests locally is difficult and this just increases the barrier to entry further